llvm/test/Feature/Repo/repo_pruning_dependents.py

```diff
@@ -0,0 +1,306 @@
+# These tests check the program repository pruning pass for the dependents.
+#
+# In the llvm-project-prepo project, the GO's digest could be generated either by the
+# RepoMeatadataGenerator pass or by the RepoObjectWriter. When a new global object (GO) is
```

Comment:
"RepoMeatadataGenerator" -> "RepoMetadataGenerator" (and in a few other places)

Reply:
Fixed, thanks.